config/environments/test.rb

```diff
@@ -12,9 +12,11 @@
   # preloads Rails for running tests, you may have to set it to true.
   config.eager_load = false
 
-  # Configure static file server for tests with Cache-Control for performance.
-  config.serve_static_files   = true
-  config.static_cache_control = 'public, max-age=3600'
+  # Configure public file server for tests with Cache-Control for performance.
+  config.public_file_server.enabled = true
+  config.public_file_server.headers = {
+    'Cache-Control' => 'public, max-age=3600'
```

Comment:
__[rubocop]__
* Style/TrailingCommaInLiteral: Put a comma after the last item of a multiline hash. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas)

<!-- Comment by [SideCI](https://sideci.com) -->